src/main/resources/application-local.yaml

```diff
@@ -36,7 +40,7 @@ spring:
 trusted:
   s2s:
     service:
-      names: ccpay_bubble,cmc,bulk_scan_payment_processor,api_gw,probate_frontend,divorce_frontend
+      names: ccpay_bubble,cmc,bulk_scan_payment_processor,api_gw,probate_frontend,divorce_frontend,payment_app
```

Comment:
change looks unrelated

Reply:
@timja Yeah this change was to sync up local prop with the actual application, but I have removed the above change as part of this pr